src/goto-programs/class_hierarchy.cpp

```diff
@@ -13,6 +13,7 @@ Date: April 2016
 
 #include "class_hierarchy.h"
 
+#include <iterator>
```

Comment:
This line was only needed to make the "AWS cbmc-windows" build happy, which would otherwise complain about `back_inserter` in `ids_from_indices`.